Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.yml

```diff
@@ -0,0 +1,96 @@
+commonfields:
+  id: Prisma Access
+  version: -1
+name: Prisma Access
+display: Prisma Access
+category: Network Security
+description: |+
+  Integrate with Prisma Access to monitor the status of the Service, alert and take actions.
+
+configuration:
+- display: Server Host or IP (e.g.,  10.1.1.9 or panorama.my.domain)
+  name: server
+  type: 0
+  required: true
+- display: API Port (e.g 443)
+  name: port
+  defaultvalue: "443"
+  type: 0
+  required: false
+- display: API Key
+  name: key
+  type: 4
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+- display: Device group - Panorama instances only (write shared for Shared location)
+  name: device_group
+  type: 0
+  required: false
+- display: Vsys - Firewall instances only
+  name: vsys
+  type: 0
+  required: false
+- display: SSH Port
+  name: sshport
+  type: 0
+  required: false
+- display: SSH Credentials for CLI
+  name: Username
+  type: 9
+  required: false
+script:
+  script: ''
+  type: python
+  commands:
+  - name: prisma-access-logout-user
+    arguments:
+    - name: user
+      required: true
+      default: true
+      description: Username to logout. (Without domain name - e.g. jsmith)
+    - name: domain
+      required: true
+      description: Domain name of the user to logout.
+    - name: computer
+      required: true
+      description: Computer name to logout.
+    outputs:
+    - contextPath: PrismaAccess.LogoutUser
+      description: LogoutUser command results
+    description: Force logout a specific user from Prisma Access
+  - name: prisma-access-query
+    arguments:
+    - name: query
+      required: true
+      description: 'Query to run. Example input: querystring limit=2000 action getGPaaSLast90DaysUniqueUsers'
```

Comment:
Is this query has well known syntax by the users? Is there a documentation for this query language that you can add a link here?

Reply:
I don't have one  right now. Asking the SME